src/cli/README.md

```diff
@@ -707,9 +707,9 @@ Done
 
 ### childsupervision interval
 
-Get the Child Supervision Interval value.
+Get the Child Supervision interval value on the child.
 
-Child supervision feature provides a mechanism for parent to ensure that a message is sent to each sleepy child within the supervision interval. If there is no transmission to the child within the supervision interval, OpenThread enqueues and sends a supervision message (a data message with empty payload) to the child. This command can only be used with FTD devices.
+Child Supervision feature provides a mechanism for parent to ensure that a message is sent to each sleepy child within the supervision interval. If there is no transmission to the child within the supervision interval, OpenThread enqueues and sends a Child Supervision Message to the child.
```

Comment:
Should the API description be updated too?

https://github.com/openthread/openthread/blob/aa7ce0d4da8d96fd7bf217d074ce0b9ea4e53c1c/include/openthread/child_supervision.h#L59